src/utils/Interfaces.ts

```diff
@@ -0,0 +1,26 @@
+export interface BorrowEligibilityResp {
```

Comment:
Cool implementation. I am thinking maybe we can have interfaces at component or page level so that we don't end up having a very large interface file. 

So maybe this interface can be in `src/.../{coponentOrPageDirectoryName}/interface.ts`, what do you think?

Reply:
Nice feedback. I've moved the interface file into the data folder where two components were referencing it